srml/balances/src/lib.rs

```diff
@@ -355,13 +371,14 @@ impl<T: Trait> Module<T> {
 		}
 	}
 
-	/// Increase TotalIssuance by Value.
+	/// Increase `TotalIssuance` by `value`.
```

Comment:
I would not document which database item is modified, especially when the name differs from the function name `issuance` vs `stake`.
Just write:
```Increase stake by `value`.```